### PR TITLE
Update faq.ja.md

### PR DESCRIPTION
**Pull request by akiyoshi**
All changes made in #122 were translated in Japanese.

### CODE REVIEW

**Comment by katabame** [comment]
@akiyoshi Could you merge changes in #134 into this pr?

**Comment by akiyoshi** [comment]
Thx! I'll merge them.

File: webroot/faq/locales/faq.ja.md [hunk 1]
```diff
@@ -141,13 +141,13 @@ A: Chia Networkは、空いているストレージ（例えばHDD）領域を
 
 + <b>Q: Chiaはオープンソースですか?</b>
 
-A: ブロック形成に関する学術論文を発表し、すべての実装コードはオープンソースになります。 私たちは<a href = "https://www.apache.org/licenses/LICENSE-2.0">Apache License</a>を使用します。テストネットだけがあってメインネットがまだない時には、コードがプロプライエタリである場合がありますが、メインネットが稼働を始まったら、確実にAppachライセンスで公開します。
+A: ブロック形成に関する学術論文を発表し、すべての実装コードはオープンソースになります。 私たちは<a href = "https://www.apache.org/licenses/LICENSE-2.0">Apache License</a>を使用します。テストネットだけがあってメインネットがまだない時には、コードがプロプライエタリである場合がありますが、メインネットが稼働を始まったら、確実にAppacheライセンスで公開します。
```
Comment (katabame) [inline webroot/faq/locales/faq.ja.md]: Correct spell is `Apache`, there's one too many `p`.

**Review by katabame — CHANGES_REQUESTED** [review]
There's still little typo

File: webroot/faq/locales/faq.ja.md (hunk 1 above)
Comment (akiyoshi) [reply webroot/faq/locales/faq.ja.md]: Oh... Sorry to bother you again.

**Review by katabame — APPROVED** [review]

**Comment by katabame** [comment]
LGTM👍

**Comment by ashadle** [comment]
Hi There, thank you so much for submitting this PR. We are rebranding the website and will have updated copy so I'm going to close out this ticket. Thank you!

**Pull request closed without merging**